src/cli/serve/serve.js

```diff
@@ -112,10 +112,17 @@ function applyConfigOverrides(rawConfig, opts, extraCliOptions) {
   if (opts.elasticsearch) set('elasticsearch.hosts', opts.elasticsearch.split(','));
   if (opts.port) set('server.port', opts.port);
   if (opts.host) set('server.host', opts.host);
-  if (opts.quiet) set('logging.quiet', true);
-  if (opts.silent) set('logging.silent', true);
-  if (opts.verbose) set('logging.verbose', true);
-  if (opts.logFile) set('logging.dest', opts.logFile);
+  if (opts.silent) {
+    set('logging.silent', true) && set('logging.root.level', 'off');
+  }
+  if (opts.verbose) {
+    // Only set logging.verbose to true when KP logging isn't configured.
+    if (!has('rawConfig.logging.root.appenders')) {
+      set('logging.verbose', true);
+    } else {
+      set('logging.root.level', 'all');
+    }
+  }
```

Comment:
This technically represents a change in behavior for folks who might have been using `--verbose` but also had set `logging.root.appenders` for some reason.

Users in that case would now have their legacy-format logs disappear and replaced with KP format logs.

This feels pretty minor, but do you think it's worth adding a comment to this effect in the release notes?

Reply:
> do you think it's worth adding a comment to this effect in the release notes?
I don't see the harm in oversharing. In fact, when we start deprecating and removing behavior folks might have gotten used to, it's better to declare it up front.